lib/schema.js

```diff
@@ -128,6 +130,8 @@ function Schema(obj, options) {
   // For internal debugging. Do not use this to try to save a schema in MDB.
   this.$id = ++id;
   this.mapPaths = [];
+  this.encryptedFields = {};
+  this._encryptionType = options?.encryptionType;
```

Comment:
Why is `_encryptionType` a separate property, instead of making `encryptionType()` get/set `options.encryptionType`? The latter would be more consistent with how Mongoose handles other options.

Reply:
Only because I didn't consider mutation the options object 🙂 .  Fixed!